ui/ui.go

```diff
@@ -2,8 +2,10 @@ package ui
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
-	"html/template"
+	htemplate "html/template"
```

Comment:
Rename these to `html_template` and `text_template`

Reply:
Good one! Will do.